Non-standard tenor volatility transformation for caplets and swaptions

sschlenkrich opened the pull request:
Hi, I would like to contribute my implementation of volatility transformation for non-standard tenor caplets and swaptions.

The methodology behind the implementation is documented here:

https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2695011
https://www.quantlib.org/slides/qlum15/schlenkrich.pdf

A unit test is added to the test suite.

Please let me know if any questions or comments.

Thanks, Sebastian

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/lballabio/QuantLib?pullRequest=575) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/lballabio/QuantLib?pullRequest=575) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/lballabio/QuantLib?pullRequest=575) it.</sub>

lballabio commented:
Thanks, Sebastian.  I've merged with master, fixed the conflicts, and added your files to the build so that they're compiled during the Travis run (see the automated checks below). I've pushed the changes to your branch.

There are errors due to the use of C++11 syntax (we're still using C++03 for backward compatibility).  May you look into them if you have some time?

sschlenkrich commented:
Yes, I will look into the failed tests.


From: Luigi Ballabio [mailto:notifications@github.com]
Sent: Mittwoch, 28. November 2018 13:18
To: lballabio/QuantLib
Cc: Schlenkrich, Sebastian; Author
Subject: Re: [lballabio/QuantLib] Non-standard tenor volatility transformation for caplets and swaptions (#575)


Thanks, Sebastian. I've merged with master, fixed the conflicts, and added your files to the build so that they're compiled during the Travis run (see the automated checks below). I've pushed the changes to your branch.

There are errors due to the use of C++11 syntax (we're still using C++03 for backward compatibility). May you look into them if you have some time?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/lballabio/QuantLib/pull/575#issuecomment-442427258>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHp0aOn78E6UVedskEgQwVrSabNplphmks5uzn7tgaJpZM4Y15nZ>.

Please do not print this document unless it is necessary, consider the environment.

d-fine GmbH: Sitz der Gesellschaft: An der Hauptwache 7, 60313 Frankfurt am Main; Amtsgericht Frankfurt am Main: HRB Nr. 48 103; Geschaeftsfuehrer: Dr. Bernd Appasamy, Dr. Mark Beinker, Christoph Belafi, Dr. Christoph Bennemann, Dr. Gernot Blum, Dr. Oliver Bohr, Dr. Christian Bormann, Dr. Christoph Burmester, Dr. Ashot Davtyan, Todor Dobrikov, Dr. Arndt Dombert, Dr. Uwe Doerr, Dr. Andreas Geyer, Nadine Gottschalk, Dr. Oliver Hein, Dr. Bodo Huckestein, Dr. Sascha Huegle, Dr. Tilman Huhne, Ulf Henning Jacobs, Dr. Andreas Keese, Dr. Moritz Kiese, Dr. Anne Kleppe, Dr. Florian Merz, Dr. Jochen Meyer, Dr. Karsten Meyer, Dr. Christian Oehler, Wolfgang Pleyer, Dr. Joern Rank, Dr. Christian Romeike, Markus von Rothkirch, Alexander Schalk, Dr. Egbert Schark, Dr. Christoph Schneggenburger, Dr. Markus Seifert, Dr. Constantin Sobiella, Artur Steiner, Dr. Roland Uhlig, Dr. Andreas Werner

This e-mail communication (and any attachment/s) is confidential and intended only for the individual(s) or entity named above and to others who have been specifically authorized to receive it. If you are not the intended recipient, please do not read, copy, use or disclose the contents of this communication to others. Please notify the sender that you have received this e-mail in error, and delete the e-mail (including any attachment/s) subsequently. This information may be subject to professional secrecy (e. g. of auditor, tax or legal advisor), other privilege or otherwise be protected by work product immunity or other legal rules.
For more information about how and why we use personal information and who to contact with any queries about this, please consult our Data Privacy Policy on https://www.d-fine.com/en_DE/privacy-policy/.
Thank you.

lballabio commented:
Thanks!
